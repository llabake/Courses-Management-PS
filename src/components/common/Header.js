import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <nav>
    <Link to="/" className="active">Home</Link>
    {' | '}
    <Link to="/courses" className="active">Courses</Link>
    {' | '}
    <Link to="/about" className="active">About</Link>
  </nav>
);

export default Header;
