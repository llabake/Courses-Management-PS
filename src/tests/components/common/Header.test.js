import React from 'react';
import { shallow } from 'enzyme';
import Header from "../../../components/common/Header";

describe('<Header/>', () => {
  const props = {
    loading: false,
    courses: [],
  };
  const wrapper = shallow(<Header {...props} />);
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
