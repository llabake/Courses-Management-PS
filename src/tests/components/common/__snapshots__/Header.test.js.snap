// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders correctly 1`] = `
<nav>
  <Link
    className="active"
    replace={false}
    to="/"
  >
    Home
  </Link>
   | 
  <Link
    className="active"
    replace={false}
    to="/courses"
  >
    Courses
    <span
      className="label label-primary"
    >
      0
    </span>
  </Link>
   | 
  <Link
    className="active"
    replace={false}
    to="/authors"
  >
    Authors
  </Link>
   | 
  <Link
    className="active"
    replace={false}
    to="/about"
  >
    About
  </Link>
</nav>
`;
